extensions/amp-story-auto-ads/0.1/amp-story-auto-ads.js

```diff
@@ -441,6 +441,7 @@ export class AmpStoryAutoAds extends AMP.BaseElement {
     const a = this.win.document.createElement('a');
     a.className = 'i-amphtml-story-ad-link';
     a.setAttribute('target', '_blank');
+    a.setAttribute('style', 'opacity:0; font-size:0; transform:scale(0)');
```

Comment:
Isn't there a `setStyle` helper for this?

Reply:
yep. Sorry didn't see this until after merge. Will make follow up.